README.rst

```diff
@@ -10,7 +10,7 @@ Images can be placed in the ``docs/images/`` folder.
 The html, js and css files can be found under ``docs/_static``. The theme used
 in these docs is the "furo" theme, which has been modified to approach a more
 Ubuntu-esque appearance. The furo documentation has `instructions on how to
-change<https://github.com/pradyunsg/furo>`_ most of the theme's functionality.
+change<https://github.com/pradyunsg/furo>most of the theme's functionality.
```

Comment:
The previous version was already a correct link in the [reStructuredText format](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html), although I have to admit that GitHub does not make a great job of rendering the link.

Maybe we should convert the README to markdown. Although it is inconsistent relative to the remaining documentation GitHub does a much better job rendering markdown. I think it is worth it, because the Repo  makes a bad impression if the README is not rendered properly. @rkratky @s-makin What do you think?